webcompat/templates/shared/footer.html

```diff
@@ -1,40 +1,44 @@
-<footer class="wc-Footer" role="contentinfo">
-  <div class="wc-UIContent wc-UIContent--footer">
-    <div class="wc-Footer-wrapper">
-      <ul class="wc-Footer-list r-ResetList">
-        <li class="wc-Footer-item">
-          <a class="wc-Footer-link wc-Footer-link--separator" href="{{ url_for('about') }}">
-            <span class="wc-Footer-link-label">About</span>
+<footer id="footer" class="grid">
+  <section class="grid-row">
+      <ul class="grid-cell x2 footer-list">
+        <li class="footer-item{% if request.url_rule.endpoint == 'about' %} active{% endif %}">
+          <a class="footer-item-link" href="{{ url_for('about') }}">
+            About
           </a>
         </li>
-        <li class="wc-Footer-item">
-          <a class="wc-Footer-link wc-Footer-link--separator" href="{{ url_for('privacy') }}">
-            <span class="wc-Footer-link-label">Privacy Policy</span>
+        <li class="footer-item{% if request.url_rule.endpoint == 'privacy' %} active{% endif %}">
+          <a class="footer-item-link" aria-label="go to privacy policy" href="{{ url_for('privacy') }}">
+            Privacy Policy
           </a>
         </li>
-        <li class="wc-Footer-item">
-          <a class="wc-Footer-link wc-Footer-link--separator" href="/humans.txt">
-            <span class="wc-Footer-link-label">Contact</span>
+        <li class="footer-item">
+          <a class="footer-item-link" aria-label="this is a txt file with all contacts" href="/humans.txt">
```

Comment:
We seem to have a `go to...` convention, so can we make this `go to an overview of contributors and contracts`?